chart/compass/values.yaml

```diff
@@ -616,7 +621,7 @@ global:
       clientIdKey: client-id
       clientSecretKey: client-secret
       oauthUrlKey: url
-      oauthTokenPath: "/secured/oauth/token"
+      oauthTokenPath: "/cert/token"
```

Comment:
It was used in 2 places - in TOKEN_PATH for the tests and also in self-reg token path. Now this is used in self-reg only and I've introduced new property(as you mentioned) tests.ordService.consumerTokenPath which is used for the test(TOKEN_PATH). It is better to be separated because they are for different flows.

Reply:
Yes, you are right, this oauthTokenPath is used for the tests, and now with my change, it is not used at all. I will revert the changes(remove the consumerPath and use the oauthTokenPath as it was).